### PR TITLE
fix: :bug: fixed transition animation in accordion to match zhweb

### PR DESCRIPTION
**Pull request by resmartiZH**
@daenub, this fixes issue #244

### CODE REVIEW

File: src/components/accordion/Accordion.js [hunk 1]
```diff
@@ -84,14 +84,19 @@ export class LeuAccordion extends LeuElement {
         <span class="label">${this.label}</span>
         <div class="plus"></div>
       </button></${unsafeStatic(hTag)}>
-      <div
-        id="content"
-        class="content"
-        aria-labelledby="toggle"
-        role="region"
+      <div 
+        id="contentwrapper"
```
Comment (daenub) [inline src/components/accordion/Accordion.js:88]: This id is never referenced. You probably should change the value of the aria-controls attribute to this id.

Move the following attributes of the "content" element to the "contentwrapper" element:
- `aria-labelledby="toggle"`
- `role="region"`

File: src/components/accordion/Accordion.js [hunk 2]
```diff
@@ -84,14 +84,19 @@ export class LeuAccordion extends LeuElement {
...
+        class="contentwrapper"  
         ?hidden=${!this.open}
       >
-        <slot name="content"></slot>
+        <div
+          id="content"
```
Comment (daenub) [inline src/components/accordion/Accordion.js:93]: This id can be removed after the `aria-controls` attribute of the toggle is updated.

**Review by daenub — CHANGES_REQUESTED** [review]
Thanks a lot for the fix!

I just saw that in firefox the divider is not visible anymore.

Otherwise it looks good to me!

File: src/components/accordion/Accordion.js (hunk 1 above)
Comment (resmartiZH) [reply src/components/accordion/Accordion.js:88]: done

File: src/components/accordion/Accordion.js (hunk 2 above)
Comment (resmartiZH) [reply src/components/accordion/Accordion.js]: done

**Comment by resmartiZH** [comment]
Thanks for the review. I fixed the bug in firefox and all the other details.

**Pull request merged**